### ISSUE TITLE
Customizable error message

### ISSUE THREAD

**Issue by edgarmueller** [eclipsesource/play-json-schema-validator#10]
The Schema validator error messages should be customizable.

**Comment by kondaurovDev** [comment]
Yes, this would be great. When there is validation error i don't want to see original jsValue because it's too big sometimes

**Comment by edgarmueller** [comment]
The initial motivation for this issue was because of i18n reasons, but we could also think about altering the general error reporting format. So, what you propose is remove the value field, since it gets too big sometimes, is that correct? I guess it would be feasible then too configure the format via the `application.conf` file? Would that work for you? Likewise, translations could be provided via `conf/message.xxx` files as described [here](https://www.playframework.com/documentation/2.5.x/ScalaI18N).
